scripts/cmake/UnityBuildSettings.cmake

```diff
@@ -0,0 +1,14 @@
+if(NOT OGS_USE_UNITY_BUILDS OR ${CMAKE_VERSION} VERSION_LESS 3.16)
+    return()
+endif()
+
+set_target_properties(BaseLib PROPERTIES UNITY_BUILD_BATCH_SIZE 8)
+set_target_properties(GeoLib PROPERTIES UNITY_BUILD_BATCH_SIZE 40)
+set_target_properties(MaterialLib PROPERTIES UNITY_BUILD_BATCH_SIZE 20)
+set_target_properties(MathLib PROPERTIES UNITY_BUILD_BATCH_SIZE 10)
+set_target_properties(MeshLib PROPERTIES UNITY_BUILD_BATCH_SIZE 20)
+# set_target_properties(ProcessLib PROPERTIES UNITY_BUILD_BATCH_SIZE 80) # breaks!
```

Comment:
Just curious: What type of error did you observe?

Reply:
I do not know anymore.. some namespace clashes.. this has to be fixed in the future but I would assign this task more to the process guys..